clients/integrations/openai/src/openai_agents_k8s_sandbox/transport.py

```
@@ -0,0 +1,218 @@
+                    message="sandbox file write failed",
+                    context={"exit_code": result.exit_code},
+                )
+        finally:
+            await self._rm_staging_best_effort(staging)
+
+    async def _rm_staging_best_effort(self, staging: str) -> None:
+        try:
+            await self._run_script('rm -f "$1"', staging)
+        except Exception:
+            # Never let cleanup replace the failure that brought us here.
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)